src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -229,7 +229,11 @@ impl DdlService for DdlServiceImpl {
 
         let version = self
             .ddl_controller
-            .run_command(DdlCommand::CreateStreamingJob(stream_job, fragment_graph))
+            .run_command(DdlCommand::CreateStreamingJob(
+                stream_job,
+                fragment_graph,
+                false,
```

Comment:
Creating a sink is also a DDL job. I think we should pass `run_in_background` to this method as well.

Reply:
Actually was planning to add it in separate PR, as mentioned in PR description:
> Later we will handle sinks and indexes as well. 

But I'm okay to add within this PR too.